cfme/tests/configure/test_docs.py

```diff
@@ -67,28 +85,28 @@ def test_links(guides, soft_assert):
 @pytest.mark.meta(blockers=[1145326, "GH#ManageIQ/manageiq:2246"])
 def test_contents(guides, soft_assert):
     """Test contents of each document."""
+    cfme_to_cf = {'5.2': '3.0', '5.3': '3.1', '5.4': '3.2'}
```

Comment:
I don't know that it changes enough to be better in one place or another, so I think it doesn't matter exactly where the mapping itself lives as much as having the function that resolves the cfme version to a cloudforms release be in utils.version